src/util/BaseLoginController.js

```diff
@@ -92,8 +92,14 @@ define(['okta', 'vendor/lib/q'], function (Okta, Q) {
     toggleButtonState: function (state) {
       var button = this.$el.find('.button');
       button.toggleClass('link-button-disabled', state);
-    }
+    },
 
+    // Event triggered at after a page is rendered along with the classname to identify the page
+    postRender: function() {
+      _.defer(_.bind(function () {
+        this.trigger('pageRenderedEvent',[this.className]);
```

Comment:
Oh, I also think we should pass the argument (className) back as an object rather than as a string. We're going to have more events, so it'd be nice to say "each event is triggered with some data; in the case of the pageRendered event, it triggers with data { page: 'primary-auth' }".